### DISCUSSION TITLE
Finding a cool name for the project

### DISCUSSION THREAD

**Discussion by donatomartin** [donatomartin/VU-CT-ai-spreadsheet-trainer#7] [Ideas]
# Finding a cool name for the project

We are taking all kinds of suggestions, try sharing fun or catchy ideas! 

![WaltwhiteBreakingbadGIF](https://github.com/user-attachments/assets/56c804ae-3054-4ef4-81a9-9e6dedef618d)

Say my name!

**Comment by mohBgz** [comment]
**XcelLearn** 😎
